docs/antctl.md

```diff
@@ -125,6 +128,52 @@ The following command prints the current feature gates:
 antctl get featuregates
 ```
 
+### Performing checks to facilitate installation process
+
+Antrea provides a utility command `antctl check` designed to perform checks that
+verify whether the Kubernetes cluster is correctly configured for installing
+Antrea, and also to confirm that Antrea has been installed correctly.
+
+#### Pre-installation checks
+
+Before installing Antrea, it's crucial to ensure that the Kubernetes cluster is
+configured properly. This can prevent potential issues that might arise during the installation
+of Antrea. To perform these pre-installation checks, Simply run the command as follows:
```

Comment:
The lines are not wrapped consistently, please wrap them at 80 chars per line.

It applies to other contents too.